encoding/encode_internal_test.go

```diff
@@ -123,3 +123,41 @@ func Test_encodeBool(t *testing.T) {
 		}
 	}
 }
+
+func Test_convertTo4Bytes(t *testing.T) {
+
+	tests := []struct {
+		operand  int
+		expected string
+	}{
+		{
+			operand:  0,
+			expected: "00000000",
+		},
+		{
+			operand:  1,
+			expected: "00000001",
+		},
+		{
+			operand:  2147483647,
+			expected: "7fffffff",
+		},
+		{
+			operand:  4294967295,
+			expected: "ffffffff",
+		},
+	}
+
+	for i, test := range tests {
+		op := test.operand
+		convertedCode := convertTo4Bytes(op)
+
+		if len(convertedCode) != 8 {
+			t.Fatalf("test[%d] - convertTo4Byte() result wrong. expected=8, got=%d", i, len(convertedCode))
```

Comment:
How about insert '\n;?
Like : test[%d] - convertTo4Byte() result wrong.`\n`expected=8, `\n`got=%d", i, len(convertedCode)
I think It has more readablity in case of error occured.

Reply:
In function convertTo4Bytes, do i block escape character ? like '\n' '\t' 'b' 
or do you mean just print out error message?

Reply:
I think @boohyunsik meant to say this..?
```
t.Fatalf("test[%d] - convertTo4Byte() result wrong.\n expected=8, got=%d", i, len(convertedCode))
```
insert `\n` between error message

Reply:
i got it. thank you~